File: int2bin.java
import java.util.Scanner;

class int2bin{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        long n = sc.nextInt(), i=0, bin=0;

        while(n!=0){
            if(n%2!=0)
                bin += Math.pow(10, i);
            i++;
            n/=2;
        }

        System.out.println(bin);
    }
}//fin class

File: int2bin.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(){

    long n, i=0, bin=0;

    scanf("%d", &n);

    while(n != 0){
        if(n%2 != 0)
            bin += pow(10,i);

        i++;
        n/=2;
    }

    printf("%d\n", bin);
}